packages/block-editor/src/components/block-mobile-floating-toolbar/nav-up-icon.js

```diff
@@ -3,7 +3,7 @@
  */
 import { SVG, Path } from '@wordpress/components';
 
-const NavigateUp = (
+const NavigateUp = ( { isRTL } ) => (
```

Comment:
Since it is only used for mobile I suggest to move the import of  I18nManager here and remove it from the floating toolbar

Reply:
I keep it according to our discussion before